test/core/consensus/validation/block_validator_test.cpp

```diff
@@ -91,7 +91,7 @@ class BlockValidatorTest : public testing::Test {
       Hash256::fromString("c30ojfn4983u4093jv3894j3f034ojs3").value();
 
   BabeSlotNumber slot_number_ = 2;
-  VRFPreOutput vrf_value_ = 1488228;
+  VRFPreOutput vrf_value_ = uint256_t_to_bytes(1488228);
```

Comment:
You should understand that VRFPreOutput is not relevant to big integer. It is not integer, it is just bytes. 
Maybe we should initialize this value differently, not to confuse others who will read the code, when they will try to understand how big integer is related to VRFPReOutput

Reply:
Do understand, didn't think of it as of an example, sorry for that, fixed.